tests/sdk/dotnet/yaml-url/Program.cs

```diff
@@ -36,7 +36,8 @@ private ConfigFile MakeConfigFile(string name, string namespaceName, Provider pr
     {
         return new ConfigFile(name, new ConfigFileArgs
         {
-            File = "https://raw.githubusercontent.com/pulumi/pulumi-kubernetes/master/tests/sdk/nodejs/examples/yaml-guestbook/yaml/guestbook.yaml",
+            // TODO: Switch back to master branch once the guestbook.yaml file is merged into mainline branch.
```

Comment:
CI for this PR presubmit will fail, since the checked in manifest in `master` is not updated yet. I'm not sure what the best way to handle this, but switching to the current commit's manifest then following up with another PR to update it to the latest in master will quickly unblock us for now.